### PR TITLE
Improve LLM layout

### PR DESCRIPTION
**Pull request by KVNLS**
### 📝 Description

Today we have some layout issues with LLM.
For example 
- we have different behavior between some sections on the wallet (Market or nft gallery)
- Momentum on iPhone overlaps the wallet header.
- On multiple pages we have some hacks to compute the right amount of space available

Before

https://github.com/LedgerHQ/ledger-live/assets/31533861/0de9b4b6-454f-4070-8a5a-6ac606f0f70d

After iOS

https://github.com/LedgerHQ/ledger-live/assets/31533861/a2578182-56c7-4914-852f-ce2fd518cdc2

With Experimental Header

https://github.com/LedgerHQ/ledger-live/assets/31533861/bd365c18-bd19-4f7f-82b1-fb31686cf9ce

Android

https://github.com/LedgerHQ/ledger-live/assets/31533861/9144b726-e659-453b-a012-7db4c374d629

Android with Experimental Header

https://github.com/LedgerHQ/ledger-live/assets/31533861/cdcae96e-efdb-4c75-bbd3-838676b2a460

I created a technical debt item for issues not solved: https://ledgerhq.atlassian.net/wiki/spaces/WXP/pages/4397269222/2023-11-28+-+LLM+Layout

### ❓ Context

- **JIRA or GitHub link**: [LIVE-9591](https://ledgerhq.atlassian.net/browse/LIVE-9591)

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-9591]: https://ledgerhq.atlassian.net/browse/LIVE-9591?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/RootNavigator/MarketNavigator.tsx [hunk 1]
```diff
@@ -19,19 +18,16 @@ export default function MarketNavigator() {
...
       };
 
   return (
     <Stack.Navigator screenOptions={stackNavigationConfig} initialRouteName={ScreenName.MarketList}>
       <Stack.Screen name={ScreenName.MarketList} component={MarketList} options={headerConfig} />
-      <Stack.Screen
-        name={ScreenName.MarketDetail}
-        component={MarketDetail}
-        options={headerConfig}
-      />
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/components/RootNavigator/MarketNavigator.tsx:34]: Not used?

Comment (KVNLS) [reply apps/ledger-live-mobile/src/components/RootNavigator/MarketNavigator.tsx:34]: Already defined in the BaseNavigator. Today when you navigate from the market to MarketDetail you keep the tabBar. When you navigate from an asset to the MarketDetail you don't have it. I discussed it with Cédric and we decided to remove the TabBar when you're on MarketDetail. So I removed this Screen definition to only keep the one in BaseNavigator.

Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/components/RootNavigator/MarketNavigator.tsx:34]: Oke nice ! :)

**Review by mcayuelas-ledger — APPROVED** [review]
Tested on iphone 14 pro

LGTM

**Pull request merged**